packages/shared/lib/src/sdk/mod.rs

```diff
@@ -18,6 +19,8 @@ pub struct Sdk {
 }
 
 #[wasm_bindgen]
+/// Sdk mostly wraps the logic of the Sdk struct members, making it a part of public API.
+/// For specifics naviagate to the corresponding modules.
```

Comment:
Minor typo: `naviagate` -> `navigate`

Could maybe be a little more clear with a comma and adjustment like following:
```
/// For more details, navigate to the corresponding modules.
```